channels/ua.m3u

```diff
@@ -217,3 +217,5 @@ http://cdn1.live-tv.od.ua:8081/wave/wave-abr/playlist.m3u8
 http://cdn1.live-tv.od.ua:8081/wave/wave-abr/wave/wave-720/playlist.m3u8
 #EXTINF:-1 tvg-id="inshiy.ua" tvg-name="інший" tvg-country="UA" tvg-language="Ukrainian" tvg-logo="" group-title="",інший (720p)
 http://cdn1.live-tv.od.ua:8081/ktkod/ktkod-abr/ktkod/ktkod/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",2x2
+http://mfe.cliptv.az/dash/2x2_SD.ism/playlist.mpd
```

Comment:
2x2 is Russian channel: https://www.lyngsat.com/tvchannels/ru/Telekanal-2x2.html